common/types/utils.d.ts

```diff
@@ -1,3 +1,4 @@
 export type StandardCallback<T> = (err?: ErrorInfo | null, result?: T) => void;
 export type ErrCallback = (err?: ErrorInfo | null) => void;
 export type PaginatedResultCallback<T> = StandardCallback<PaginatedResult<T>>;
+export type Modify<T, R> = Omit<T, keyof R> & R;
```

Comment:
If you're adding a brand new type here, which I think you are, then this would be a great opportunity to document what it does / conveys. It's quite opaque.